src/dso_api/settings.py

```diff
@@ -176,7 +221,7 @@
     },
     "root": {"level": "INFO", "handlers": ["console"]},
     "loggers": {
-        # "django.db.backends": {"level": "DEBUG", "handlers": ["console"]},
+        "django": {"handlers": ["console"], "level": DJANGO_LOG_LEVEL, "propagate": False},
         "dso_api": {"handlers": ["console"], "level": "DEBUG", "propagate": False},
```

Comment:
Unrelated to current PR (not totally, but does not have to be fixed now): do we really need `DEBUG` logging on production?

Reply:
Changed default level to `INFO` and made it configurable by means of an environment variable.